### ISSUE TITLE
Adding 'remove widget' label as multilangual message.

### ISSUE THREAD

**Issue by miladhashemi** [os-js/osjs-widgets#11]
Hi,
I'm developing a widget that supports multilangual. but the 'remove widget' label added directly to widget.
https://github.com/os-js/osjs-widgets/blob/69174f5256b6f49af378e8dd7bcec1ed80329bd7/src/widget.js#L314

Must be:
```js
const {translate: _, translatable} = this.core.make('osjs/locale');
.
.
label: _('LBL_REMOVE_WIDGET'),
```

Or adding seperate locales.js for widget:
```js
const __ = translatable(translations);
.
.
label: __('LBL_REMOVE_WIDGET'),
```

> osjs-widgets v3.0.7
> osjs-client v3.1.66

**Comment by andersevenrud** [comment]
Feel free to make a PR on this if you're already developing widgets :)

I might not  have time before next week to look at this.

**Comment by andersevenrud** [comment]
I found a bit of time to do this. Released with the new 3.0.8 release of `@osjs/widgets`.

**Comment by andersevenrud** [comment]
Also, I've released a new update (3.0.9) that supports adding proper titles to the contextmenu (and other things) with a new `static metadata(core)` method in the `Widget` class.

Manual was updated with an example: https://manual.os-js.org/tutorial/widget/

**Comment by miladhashemi** [comment]
Thank U.
